### PR TITLE
ConFoo 2015 early bird news

### PR DESCRIPTION
**Pull request by afilina**
Added a news entry and image.

### CODE REVIEW

File: archive/entries/2014-12-12-1.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <published>2014-12-12T18:17:52+00:00</published>
+  <updated>2014-12-12T18:17:52+00:00</updated>
+  <default:finalTeaserDate xmlns="http://php.net/ns/news">2015-02-18</default:finalTeaserDate>
+  <category term="conferences" label="Conference announcement"/>
+  <link href="http://php.net/conferences/index.php#id2014-12-12-1" rel="alternate" type="text/html"/>
+  <default:newsImage xmlns="http://php.net/ns/news" link="http://confoo.ca" title="Become a master: improve your skills at ConFoo">confoo-2015.jpg</default:newsImage>
+  <link href="http://confoo.ca" rel="via" type="text/html"/>
+  <content type="xhtml">
+    <div xmlns="http://www.w3.org/1999/xhtml">
+     <p>We want you to learn as much as possible during the three days of conference. We do that through quality and variety of both content and speakers, as well as creating a fun and friendly atmosphere.<p>
```
Comment (salathe) [inline archive/entries/2014-12-12-1.xml]: This line ends with a broken tag... `<p>` should be `</p>`.

**Comment by salathe** [comment]
Hi Anna, can you fix the broken XML (bad closing paragraph tag) and rebase against current master? Then it looks good to merge.

**Comment by Stelian** [comment]
Hello Anna, can you please merge master in? There are some conflicts that need to be resolved. I will merge this as soon as that is out of the way.

**Comment by afilina** [comment]
I created a separate pull request with everything fixed:
https://github.com/php/web-php/pull/57

Anna Filina
1-888-392-0909 ext. 102

**Comment by php-pulls** [comment]
**Comment on behalf of stelianm at php.net:**

Was replaced by PR !57

**Pull request closed without merging**